lib/vsc/install/shared_setup.py

```diff
@@ -1524,6 +1524,7 @@ def action_target(package, *args, **kwargs):
     """
     install_requires = [
         'setuptools',
+        'future'
```

Comment:
should be added to vsc-base, not vsc-install? or is it also used here
it has to be added to vsc-base, because i'd rather have no runtime dependecies on vsc-install